src/codeGenerators/circuit/zokrates/toCircuit.ts

```diff
@@ -113,8 +113,11 @@ function codeGenerator(node: any) {
     case 'IndexAccess':
       return `${codeGenerator(node.baseExpression)}_${codeGenerator(node.indexExpression)}`;
 
+    case 'TupleExpression':
+      return `(${node.components.map(codeGenerator).join(` `)})`;
+
```

Comment:
Quite a few bits here are to add the solidity node type `TupleExpression`, which covers pretty much everything in brackets